src/pseudopeople/interface.py

```diff
@@ -30,6 +30,19 @@ def _generate_form(
         Noised form data in a pd.DataFrame
     """
     configuration_tree = get_configuration(configuration)
+    if source is None:
+        # TODO: hard-coding the .parquet extension for now. This will go away
+        #  once we only support passing the root directory of the data.
+        # TODO: we should save outputs of the simulation with filenames that are
+        #  consistent with the names of the forms if possible.
+        form_file_name = {
+            FORMS.acs.name: "household_survey_observer_acs",
+            FORMS.cps.name: "household_survey_observer_cps",
+            FORMS.tax_w2_1099.name: "tax_w2_observer",
+            FORMS.wic.name: "wic_observer",
```

Comment:
What about the rest of the forms?

Reply:
They can use the default value of the `get` below.

Reply:
Ah, totally missed that